labgrid/driver/qemudriver.py

```diff
@@ -315,21 +315,21 @@ def monitor_command(self, command, arguments={}):
                 "Can't use monitor command on non-running target")
         return self.qmp.execute(command, arguments)
 
-    def _add_port_forward(self, proto, local_address, local_port, remote_address, remote_port):
+    def _add_port_forward(self, proto, local_address, local_port, remote_address, remote_port, netdev):
         self.monitor_command(
             "human-monitor-command",
-            {"command-line": f"hostfwd_add {proto}:{local_address}:{local_port}-{remote_address}:{remote_port}"},
+            {"command-line": f"hostfwd_add {netdev} {proto}:{local_address}:{local_port}-{remote_address}:{remote_port}"},
         )
 
-    def add_port_forward(self, proto, local_address, local_port, remote_address, remote_port):
-        self._add_port_forward(proto, local_address, local_port, remote_address, remote_port)
-        self._forwarded_ports[(proto, local_address, local_port)] = (proto, local_address, local_port, remote_address, remote_port)
+    def add_port_forward(self, proto, local_address, local_port, remote_address, remote_port, netdev=""):
+        self._add_port_forward(proto, local_address, local_port, remote_address, remote_port, netdev)
+        self._forwarded_ports[(proto, local_address, local_port)] = (proto, local_address, local_port, remote_address, remote_port, netdev)
 
-    def remove_port_forward(self, proto, local_address, local_port):
-        del self._forwarded_ports[(proto, local_address, local_port)]
+    def remove_port_forward(self, proto, local_address, local_port, netdev=""):
+        del self._forwarded_ports[(proto, local_address, local_port, netdev)]
```

Comment:
The dictionary's key does not include the netdev, see line 326. How is that supposed to work?